test/gui/tst_syncing/test.feature

```diff
@@ -327,4 +327,63 @@ Feature: Syncing files
+        Given user "Alice" has set up a client with default settings
+        When user "Alice" creates a file "<filename>" with the following content inside the sync folder
+            """
+            test content
+            """
+        And the user waits for file "<filename>" to be synced
+        Then as "Alice" file "<filename>" should exist on the server
+        Examples:
+            | filename                                                                                                                                                                                                                     |
+            | thisIsAVeryLongFileNameToCheckThatItWorks-thisIsAVeryLongFileNameToCheckThatItWorks-thisIsAVeryLongFileNameToCheckThatItWorks-thisIsAVeryLongFileNameToCheckThatItWorks-thisIsAVeryLongFileNameToCheckThatItWorks-thisIs.txt |
+
+
+    Scenario Outline: Filename with long name(>228 characters) is blacklisted
+        Given user "Alice" has set up a client with default settings
+        When user "Alice" creates a file "<filename>" with the following content inside the sync folder
+            """
+            test contents
+            """
+        And the user clicks on the activity tab
+        And the user selects "Not Synced" tab in the activity
+        And the user waits until at least a file is blacklisted
+        Then the file "<filename>" should be blacklisted
+        Examples:
+            | filename                                                                                                                                                                                                                              |
+            | aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBrownFoxJumpsOverAVeryLazyDog-aQuickBro.txt |
+
+
+    Scenario: File of 1 GB size sync successfully
+        Given user "Alice" has set up a client with default settings
+        When user "Alice" creates a file "newfile.txt" with size "1GB" inside the sync folder
+        And the user waits for file "newfile.txt" to be synced
+        Then as "Alice" file "newfile.txt" should exist on the server
+
+
+    Scenario: File with spaces in the name can sync
+        Given user "Alice" has set up a client with default settings
+        When user "Alice" creates a file "file with space.txt" with the following content inside the sync folder
+            """
+            test contents
+            """
+        And the user waits for file "file with space.txt" to be synced
+        Then as "Alice" file "file with space.txt" should exist on the server
+
+
+    Scenario: Folders with 500 files can sync successfully
+        Given user "Alice" has set up a client with default settings
+        And the user has created a folder "folder1" with "500" files each of size "1048576" bytes in the file system
```

Comment:
Comment can be added.
the step creates folder outside of the sync folder.

Other LGTM